main/supervisor/APKBUILD

```diff
@@ -1,14 +1,14 @@
 # Contributor: Peter Bui <pnutzh4x0r@gmail.com>
 # Maintainer:
 pkgname=supervisor
-pkgver=3.3.5
+pkgver=4.0.2
 pkgrel=0
 pkgdesc="A system for controlling process state under UNIX"
 url="http://supervisord.org"
 arch="noarch"
 license="BSD"
-depends="python2 py-meld3 py-setuptools"
-makedepends="$depends_dev"
+depends="python3 py3-meld3"
+makedepends="python3-dev"
```

Comment:
can be py3-setuptools

Reply:
Not sure why it may need setup tools? please elaborate

Reply:
`python3`currently includes setup tools module. Perhaps it should not if we want to continue having `py3-setuptools` as a distinct package?